storage/store.go

```diff
@@ -1584,6 +1591,35 @@ func (s *Store) GroupLocker() sync.Locker {
 	return &s.raftGroupLocker
 }
 
+// CanApplySnapshot implements the multiraft.Storage interface.
+func (s *Store) CanApplySnapshot(rangeID roachpb.RangeID, snap raftpb.Snapshot) bool {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	if r, ok := s.replicas[rangeID]; ok && r.isInitialized() {
+		// We have the range and it's initialized, so let the snapshot
+		// through. We'll decide later if it's newer or older than our
```

Comment:
Actually that decision is internal to raft, so we won't get here unless raft believes this snapshot is newer than what we have. I'll remove this sentence.